www/shared/styles/base/font-faces.css

```diff
@@ -4,9 +4,13 @@
    directory.
    ================================================================================ */
 
+/* stylelint-disable */
```

Comment:
Disable only the rule in question: `/* stylelint-disable scale-unlimited/declaration-strict-value */`